core/spec/lib/spree/money_spec.rb

```diff
@@ -268,4 +268,32 @@
       end
     end
   end
+
+  describe "<=>" do
```

Comment:
What about testing that `Comparable` is included? This way, if we remove that module, we'll be sure that some test will fail.